public/styles/interface.css

```diff
@@ -61,6 +61,7 @@ input:focus {
 
 .pulse {
   animation: pulse 3s linear infinite;
+  filter: blur(0.6px);
```

Comment:
makes the text slightly blurry, to add to the CRT look. lemme know if it's too much